SETUP.md

```diff
@@ -9,6 +9,7 @@ Setup is supported on Linux and macOS, but [**not on Windows**](https://github.c
 
 First, you must configure two environment variables (`OWNER` and `REPO`).
 These variables specify the GitHub repository for the manuscript (i.e. `https://github.com/OWNER/REPO`).
+Both variables are case-sensitive in some of the setup steps.
```

Comment:
I wonder how helpful this is compared to saying:

1. things generally are case sensitive in this script
2. make sure to use your username capitalization from GitHub.

Reply:
I could do that instead.  I believe the repo is also case sensitive.  Should I keep the comment here or move it earlier in the guide?

Reply:
If I go to https://github.com/Greenelab/Manubot-rootstock, GitHub does not redirect to the canoical capitalization. However, if you click on any GitHub links, it contains the all lowercase capitalization we set. I wonder what the best way to describe this... "use the capitalization shown on GitHub"

At this point however, REPO does not exist... it gets created in the guide.

What about:

> Make sure to use the same casing for your username as gets displayed on GitHub. In general, assume that all commands in this setup are case sensitive.

Reply:
Oh yes, I see.  I was testing Travis and GitHub repo case-sensitivity on existing repos.  Whatever case is provided here will be valid, but the user will then be stuck with it.

I revised your suggested phrasing in 5e2f179.